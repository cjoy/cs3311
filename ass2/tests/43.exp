Allison Memari (3306815)
Suggestions for study in 12s1

Course   Title                 UOC Requirement
-------- -----                ---- -----------
ENGG1000 Engineering Design      6 Level 1 Core Courses
COMP2041 Software Constructio    6 Level 2 Core Courses
COMP2111 System Modelling and    6 Level 2 Core Courses
COMP2121 Microprocessors & In    6 Level 2 Core Courses
COMP2911 Eng. Design in Compu    6 Level 2 Core Courses
Free.... Free Electives (many    6 Free elect
GenEd... General Education (m    6 GE
