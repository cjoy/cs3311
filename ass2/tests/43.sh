# Student 3306815 studied 3648, SENGA1 starting in 11s1
php advisor 3306815 11s2
