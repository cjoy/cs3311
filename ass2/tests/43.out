Allison Memari (3306815)
Suggestions for study in 12s1

168 170Course   Title                 UOC Requirement
-------- -----                ---- -----------
